Gopkg.toml

```diff
@@ -20,35 +20,116 @@ required = [
 
 ignored = [
   # Non-existent package used by a toy program in c-deps/protobuf.
-  "github.com/google/protobuf/examples/tutorial"
+  "github.com/google/protobuf/examples/tutorial",
 ]
 
-# https://github.com/cockroachdb/raven-go/pulls/1 avoids banned go/build import.
-[[constraint]]
-  name = "github.com/getsentry/raven-go"
-  revision = "221b2b44fb33f84ed3ea13f3aed62ff48c85636b"
-  source = "https://github.com/cockroachdb/raven-go"
-
 # The collation tables must never change.
 [[constraint]]
   name = "golang.org/x/text"
   revision = "470f45bf29f4147d6fbd7dfd0a02a848e49f5bf4"
 
+# https://github.com/cenkalti/backoff/commit/1f265ab
+[[constraint]]
+  name = "github.com/cenk/backoff"
+  branch = "master"
+
+# https://github.com/chzyer/readline/pull/121
+[[constraint]]
+  name = "github.com/chzyer/readline"
+  source = "https://github.com/cockroachdb/readline"
+  branch = "master"
+
 # https://github.com/docker/docker/issues/29362
 [[constraint]]
   name = "github.com/docker/docker"
   revision = "7248742ae7127347a52ab9d215451c213b7b59da"
 
-# TODO(dt,mjibson): API changed
+# https://github.com/getsentry/raven-go/pull/139
+[[constraint]]
+  name = "github.com/getsentry/raven-go"
+  source = "https://github.com/cockroachdb/raven-go"
+
+# Used for benchmarks, should be recent.
 [[constraint]]
-  name = "github.com/Azure/azure-sdk-for-go"
-  revision = "8dd1f3ff407c300cff0a4bfedd969111ca5a7903"
+  name = "github.com/go-sql-driver/mysql"
+  branch = "master"
 
+# Need https://github.com/gogo/protobuf/commit/9aee220
 [[constraint]]
-  name = "github.com/chzyer/readline"
-  source = "https://github.com/cockroachdb/readline"
+  name = "github.com/gogo/protobuf"
   branch = "master"
 
+# https://github.com/golang/dep/commit/cbc263f introduces a dependency on
+# github.com/go-yaml/yaml which is not compatible with our dependency on
```

Comment:
ugh. we should consider a .bin/vendor or something, so our `vendor` dir only has to satisfy our actual deps, and not all the transitive deps of all our dev tools too.